cfgov/jobmanager/migrations/0002_auto_20160809_1619.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class MySQLOnlyRunSQL(migrations.RunSQL):
```

Comment:
Unfortunately Django doesn't provide a way to disable migrations per-backend. The character set and collation alterations that need to happen to this table aren't supported properly in sqlite, so this migration needs to be skipped when running the tests.